scripts/staking-set-initial-pools-metadata.js

```diff
@@ -0,0 +1,40 @@
+const { ethers } = require('hardhat');
+const { addresses, StakingPool, StakingPoolFactory } = require('@nexusmutual/deployments');
+
+async function main() {
+  const factory = await ethers.getContractAt(StakingPoolFactory, addresses.StakingPoolFactory);
+  const stakingPoolCount = (await factory.stakingPoolCount()).toNumber();
+  const stakingPoolIds = new Array(stakingPoolCount).fill('').map((_, i) => i + 1);
+  const cover = await ethers.getContractAt('Cover', addresses.Cover);
+
+  const ipfsHashes = [];
+
+  for (const poolId of stakingPoolIds) {
+    const stakingPoolAddress = await cover.stakingPool(poolId);
+    const stakingPool = await ethers.getContractAt(StakingPool, stakingPoolAddress);
+
+    const filter = stakingPool.filters.PoolDescriptionSet();
+    const events = await stakingPool.queryFilter(filter, 0, 'latest');
+
+    const hash = events.length > 0 ? events[events.length - 1].args.ipfsDescriptionHash : '';
+    console.log(`Pool ${poolId}: ${hash}`);
+
+    ipfsHashes.push(hash);
+  }
+
+  const encodedData = ethers.utils.defaultAbiCoder.encode(['string[]'], [ipfsHashes]);
+  const functionSignature = ethers.utils.id('setInitialMetadata(string[])').slice(0, 10);
+  const data = functionSignature + encodedData.slice(2);
+
+  console.log('Tx details:');
+  console.log('from: [any ab member]');
+  console.log('to:', addresses.StakingProducts);
+  console.log('msg.data', data);
```

Comment:
We should simulate this to tenderly then we'll send the successful simulated tx to the the AB member so they can copy the tx.data to execute themselves.

This helpers file has `simulateTransaction` function that we can use:
https://github.com/NexusMutual/smart-contracts/blob/release-candidate/scripts/products/helpers.js

Reply:
manual simulation should do it. the ab will copy the tx data manually anyway

Reply:
ok, i'll add steps in the release docs for the migration plan - including this manual tenderly simulation